samples/04_gis_analysts_data_scientists/identifying-country-names-from-incomplete-house-addresses.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>The data licenses can be found in <code>data/country-classifier/LICENSE.txt</code>. -&gt; Replace saying the data will be downloaded from an ArcGIS Item.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>For the dataset we have picked up for the sample, mentioning the data license is required. What would be the best way to do so here?</p>

Reply:
<p>I get it now, its the relative location within the data archive. I misunderstood it earlier</p><p> </p>